src/components/DropdownV2/DropdownV2.js

```diff
@@ -141,7 +154,7 @@ export default class DropdownV2 extends React.Component {
                     isActive={selectedItem === item}
                     isHighlighted={highlightedIndex === index}
                     {...getItemProps({ item, index })}>
-                    {itemToString(item)}
+                    <ItemToElement key={itemToString(item)} {...item} />
```

Comment:
HI @justinpurcell1 👋 when `item` is a string, the spread operator here seems to make it an array of characters (ref: https://github.com/IBM/carbon-components-react/issues/1586). Do you want to use this render-element approach here only if `itemToElement` is given?